.github/scripts/check-folders.js

```diff
@@ -70,6 +88,7 @@ async function main() {
       return acc.concat([...folders])
     }, [])
 
+  // TODO ??
```

Comment:
It's transforming the shape to return something that can be easy to use from GitHub Actions if conditionals

Reply:
Looks like 
```
Object {
  "binaries": true,
  "bundlers": true,
  "core-features": true,
  "databases": true,
  "databases-macos": true,
  "dataproxy": true,
  "docker": true,
  "engines": true,
  "frameworks": true,
  "libraries": true,
  "migrate": true,
  "node": true,
  "os": true,
  "packagers": true,
  "platforms": true,
  "platforms-serverless": true,
  "platforms-serverless-vercel": true,
  "process-managers": true,
  "test-runners": true,
}
```
See https://github.com/prisma/ecosystem-tests/blob/dev/.github/workflows/detect-jobs-to-run.test.js

Reply:
```suggestion
  // Transform the shape to return something that can be easy to use from GitHub Actions if conditionals
```